liblwgeom/lwgeom_log.h

```diff
@@ -126,7 +126,11 @@
  * For debugging, use LWDEBUG() or LWDEBUGF().
  * @ingroup logging
  */
-void lwnotice(const char *fmt, ...) __attribute__ ((format (printf, 1, 2)));
+void lwnotice(const char *fmt, ...)
+#ifdef __GNUC__
```

Comment:
Also available with CLANG https://clang.llvm.org/docs/AttributeReference.html#format (but with __printf__ in the doc). I cannot test this week.

Reply:
I could be wrong for specific cases, but I believe `__GNUC__` is also defined on clang - as it is meant to be compatible with GCC in almost every way. See this rather humorous SO post: https://stackoverflow.com/questions/38499462/how-to-tell-clang-to-stop-pretending-to-be-other-compilers

I still need to do some more testing as well, though. Let me know if you uncover anything I missed in your own testing.